DoSomething/quasar
# Updates DBT docs to use Prod schema but with schema only access user.

#### What's this PR do?
Updates DBT docs to to use Prod schema based on a read-only schema access user so we don't have odd schema's like `dbt_schema` and our doc diffs aren't so crazy between different devs updating them.

#### How should this be manually tested?
Setup the DBT Docs user profile in your `profiles.yml` file.

Checkout branch and run `dbt docs generate` using the `docs` profile and see if you get any git diffs!

If no diffs, things are working, but make a final inspection via `dbt docs serve` to make sure everything looks good.

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/169357116
